packages/dashboard/src/features/ai/helpers.ts

```
@@ -220,7 +219,25 @@ export const formatPrice = (price?: number): string => {
+  model.inputPriceLabel ?? formatPrice(model.inputPrice);
+
+export const formatOutputPrice = (model: Pick<ModelOption, 'outputPrice' | 'outputPriceLabel'>) =>
+  model.outputPriceLabel ?? formatPrice(model.outputPrice);
+
 // Format modality for display
 export const formatModality = (modality: string): string => {
   return modality.charAt(0).toUpperCase() + modality.slice(1);
 };
+
+export const formatReleasedDate = (created?: number): string => {
+  if (created == null) {
```

ESLint

[error] Expected '===' and instead saw '=='.

(eqeqeq)